app/facades/food_facade.rb

```diff
@@ -0,0 +1,17 @@
+class FoodFacade
+    def self.food_search(keyword)
```

Comment:
Nice job setting up the facade. How could we make one call and get both the keyword search results and a total hits?